driver/src/main/java/org/neo4j/driver/internal/cluster/Rediscovery.java

```diff
@@ -101,4 +98,12 @@ public ClusterComposition lookupRoutingTable( ConnectionPool connections, Routin
             }
         }
     }
+
+    private void assertRouterIsNotEmpty( int size )
```

Comment:
Method can be called `assertHasRouters()` and take a `RoutingTable` as an argument